haiku-games/bebattle/bebattle-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Battle is a fun board strategy game"
+DESCRIPTION="BeBattle is a fun board strategy game."
```

Comment:
SUMMARY should not start with the program's name, both here should have a different string